javascript/javascript1/week2/main.js

```diff
@@ -83,3 +83,19 @@ function getEventWeekday(eventdayNumber) {
 
 today("Sunday");
 getEventWeekday(8);
+
+// Weather wear
+function todaysTemperature(temperature) {
+  if (temperature <= 0) {
+    console.log("It might be snowing!! wear your snow jacket and snow boot.");
+  } else if (temperature >= 1 && temperature < 6) {
+    console.log("It's pretty cold!! get ready with winter wear.");
+  } else if (temperature >= 6 && temperature < 18) {
+    console.log(
+      "It's not too cold not too hot, a light winter jacket would be fine."
+    );
+  } else {
+    console.log(" It's pretty warm!! T-shirt and shorts would be perfect.");
+  }
+}
+todaysTemperature(8);
```

Comment:
Clean implementation, great work 👌

**Food for thought** 
For the if else statements, the chronology of the conditions matters. Try to see what happens if you modify your conditions slightly as suggested below; 

if (temperature <= 0) {
    console.log("It might be snowing!! wear your snow jacket and snow boot.");
  } else if (temperature < 6) {
    console.log("It's pretty cold!! get ready with winter wear.");
  } else if (temperature < 18) {
    console.log(
      "It's not too cold not too hot, a light winter jacket would be fine."
    );
  } else {
    console.log(" It's pretty warm!! T-shirt and shorts would be perfect.");
  }
  
  Does that provide the same result as your original implementation? And why might that be? 😊
  Reminder 💡=> consider whether you want your function to return values, in your implementation you are logging.

Reply:
yes, you are right," For the if-else statements, the chronology of the conditions matters." I changed the conditions accordingly, it works fine as the conditions are ordered chronologically. But if not then will return the wrong value, so to be on the safe side I added the extra condition :).